game/handler.go

```diff
@@ -1031,13 +1033,52 @@ func handleReSchedule(w ResponseWriter, r Request) error {
 	return nil
 }
 
+func handleTestTrueSkillRateGameResults(w ResponseWriter, r Request) error {
+	if !appengine.IsDevAppServer() {
+		return HTTPErr{"unauthorized", http.StatusForbidden}
+	}
+	return handleTrueSkillRateGameResults(w, r)
+}
+
 func handleTestReapInactiveWaitingPlayers(w ResponseWriter, r Request) error {
 	if !appengine.IsDevAppServer() {
 		return HTTPErr{"unauthorized", http.StatusForbidden}
 	}
 	return handleReapInactiveWaitingPlayers(w, r)
 }
 
+func handleTrueSkillRateGameResults(w ResponseWriter, r Request) error {
+	ctx := appengine.NewContext(r.Req())
+
+	log.Infof(ctx, "handleTrueSkillRateGameResults(..., ...)")
+
+	iterator := datastore.NewQuery(gameResultKind).Filter("Private=", false).Filter("TrueSkillRated=", false).Order("CreatedAt").Run(ctx)
```

Comment:
Order should really be "finished at" I think?

Reply:
Unless applying TrueSkill is associative?

Reply:
These are GameResults, not Games, so they are created when the game is finished.